src/components/molecules/Settings/Navigation/index.tsx

```diff
@@ -66,21 +70,25 @@ const Navigation: React.FC<Props> = ({ team, project }) => {
 };
 
 const Wrapper = styled.div`
-  width: 250px;
+  width: 100%;
+  height: 100%;
   display: flex;
   flex-direction: column;
   align-items: center;
-  padding: 0;
-  margin-left: 10px;
-  margin-right: 80px;
+  padding: 32px;
+  box-sizing: border-box;
 `;
 
 const NavigationList = styled.ul`
   width: 100%;
   margin: 0;
   padding: 0;
-  font-size: ${fonts.sizes.l}px;
-  font-weight: bold;
+  font-size: ${fonts.sizes.m}px;
+`;
+
+const Line = styled.hr`
```

Comment:
We have a Divider atom component. If that is useable here think about using that ;)

Reply:
ok. sure